donate.html

```diff
@@ -1637,19 +1734,19 @@ <h4>Shipping Address</h4>
     };
 
     // Use this to hide fields as appropriately
-    const toggleShowFields = (fields, show) => {
-      $.each(fields, (index, field)=>{
+    const toggleShowFields = function (fields, show) {
+      $.each(fields, function (index, field) {

```

Comment:
Were these updates from arrow functions done by a linter? It looks pretty consistent

Reply:
They weren't I had to go through and change them as arrow functions don't work on older browsers.